packaging/nnstreamer.spec

```diff
@@ -9,6 +9,15 @@
 %define		flatbuf_support 1
 %define		protobuf_support 1
 %define		nnfw_support 1
+%define		enable_test 1
+
+%if ("%{_vd_cfg_product_type}" == "AUDIO") || ("%{_vd_cfg_product_type}" == "TV") || ("%{_vd_cfg_product_type}" == "LFD") || ("%{_vd_cfg_product_type}" == "HTV") || ("%{_vd_cfg_product_type}" == "AV") || ("%{_vd_cfg_product_type}" == "IWB") || ("%{_vd_cfg_product_type}" == "WALL")
```

Comment:
Or... simply..
```
%if "%{?profile}" == "tv"
```

Reply:
We can't. Public Tizen TV profile is different from VD product.
I don't know it is the right approach but other packages use this way.
I will check it later.